packages/xl-odt-exporter/src/odt/odtExporter.tsx

```diff
@@ -0,0 +1,353 @@
+import {
+  Block,
+  BlockNoteSchema,
+  BlockSchema,
+  COLORS_DEFAULT,
+  Exporter,
+  ExporterOptions,
+  InlineContentSchema,
+  StyleSchema,
+  StyledText,
+} from "@blocknote/core";
+import { BlobWriter, TextReader, ZipWriter, BlobReader } from "@zip.js/zip.js";
+import { renderToString } from "react-dom/server";
+import stylesXml from "./template/styles.xml?raw";
+import { loadFileBuffer } from "@shared/util/fileUtil.js";
+import { getImageDimensions } from "./imageUtil.js";
+
+export class ODTExporter<
+  B extends BlockSchema,
+  S extends StyleSchema,
+  I extends InlineContentSchema
+> extends Exporter<
+  B,
+  I,
+  S,
+  React.ReactNode,
+  React.ReactNode,
+  Record<string, string>,
+  React.ReactNode
+> {
+  private automaticStyles: Map<string, React.ReactNode> = new Map();
+  private pictures: Map<
+    string,
+    {
+      file: Blob;
+      fileName: string;
+      height: number;
+      width: number;
+    }
+  > = new Map();
```

Comment:
Not immediately obvious what this is key-ed by.

I'd also have different naming, but just a nit:
```suggestion
  private images = new Map<
    string,
    {
      content: Blob;
      name: string;
      height: number;
      width: number;
    }
  >();
```